docs/satellite-clone.md

```diff
@@ -1,19 +1,22 @@
-## Cloning a Satellite server
+## Cloning or Migrating a Satellite server
 
  **Important Notes:**
 
-  - The playbook will update the cloned Satellite hostname to match the hostname of the original Satellite from which the backup is generated.
-  - DHCP, DNS, and TFTP will be disabled during the install to avoid configuration errors. If you want to use provisioning on the cloned Satellite, you will have to manually re-enable these settings.
+  - The playbook will update the cloned or migrated Satellite hostname to match the hostname of the original Satellite from which the backup is generated.
+  - DHCP, DNS, and TFTP will be disabled during the install to avoid configuration errors. If you want to use provisioning on the cloned Satellite, you will have to manually re-enable these settings after the playbook run.
   - The playbook will reset the admin password to "changeme".
-  - The playbook installs Satellite and may may take a while to complete.
-  - Make sure that the destination node has the same OS version as the original Satellite from which the backup data was generated.
+  - The playbook run may may take a while to complete.
+  - Destination node OS:
+    - For cloning: The destination node must have the same OS version as the original Satellite from which the backup data was generated.
+    - For Migration: The destination node must be rhel7.
```

Comment:
what does migration here mean? Can you specify RHEL 6 to 7 migration? You could also include any details about migrating a server from rhel 6 to 7 here. I.E "A backup for a RHEL 6 satellite can be cloned to a RHEL 7 machine. Use the satellite-migration variable as detailed below"